CorsixTH/Lua/dialogs/resizables/cheats_dialog.lua

```diff
@@ -62,8 +62,13 @@ local col_cheated_yes = {
 function UICheats:UICheats(ui)
   self.cheats = ui.hospital.hosp_cheats
   self.cheat_list = ui.hospital.hosp_cheats.cheat_list
+  self.resize_redraw = UICheats
 
-  self:UIResizable(ui, 300, 200, col_bg)
+  local width, header_indent, header_width, second_col = 300,  20, 260, false
+  if 180 + 20 * #self.cheat_list > ui.app.config.height then -- Cheats list will not fit in one column
+    width, header_indent, header_width, second_col     = 560, 100, 360, 280
```

Comment:
Shouldn't it set `second_col` ?

Reply:
Ah, my wrong, it does. It doesn't use `true`  though, that's somewhat confusing.